src/shared/config.ts

```diff
@@ -46,6 +60,55 @@ export interface ServiceConfig {
     SHOW_HIDDEN: boolean;
     COLOR: boolean;
   };
+  SHARED: {
+    PEER_ENDPOINT: string;
+    ALS_ENDPOINT: string;
+    QUOTES_ENDPOINT: string;
+    TRANSFERS_ENDPOINT: string;
+    BULK_TRANSFERS_ENDPOINT: string;
+    DFSP_ID: string;
+    JWS_SIGN: boolean;
+    JWS_SIGNING_KEY: PathLike | Buffer;
+    WSO2_AUTH: {
+      staticToken: string;
+      tokenEndpoint: string;
+      clientKey: string;
+      clientSecret: string;
+      refreshSeconds: number;
+    };
+    TLS: {
+      inbound: {
+        mutualTLS: {
```

Comment:
I wonder if we can use the power of typescript to better express this configuration.

Maybe something like (and this is rough so forgive me)

```ts

type TLSCredential = {
  ca: Array<Buffer>;
  cert: Array<Buffer>
  key: Array<Buffer>
}

type Option<T> =  T | null;

const mySetCred: Option<TLSCredential> = {
  ca: [],
  cert: [],
  key: [],
}
```

Reply:
Yes absolutely. I plan to break it down into smaller chunks.
Wanted to get feedback on the way I'm loading file data into env since it feels a bit weird...(I should've stated that)
Thanks for the feedback.

Reply:
if possible, please use `interface` instead of `type`